test/system/030-run.bats

```diff
@@ -1233,4 +1233,18 @@ search               | $IMAGE           |
     done < <(parse_table "$tests")
 }
 
+@test "podman run umask" {
+    default_umask="0022"
+    umask="0017"
+    run_podman run -q $IMAGE grep Umask /proc/self/status
+    is "$output" "Umask:.*$default_umask" "default_umask should not be modified"
+
+    run_podman run -q --umask $umask $IMAGE grep Umask /proc/self/status
+    is "$output" "Umask:.*$umask" "umask should be modified"
+    run_podman run -q -d --umask $umask $IMAGE sleep inf
+    cid=$output
+    run_podman exec $cid grep Umask /proc/self/status
+    is "$output" "Umask:.*$umask" "exec umask should match container umask"
```

Comment:
I'd also like to see something like `run_podman exec $cid touch /foo && stat -c '%a' /foo`, to actually **confirm** umask.